File: Go/1096.go
// Sequemcia IJ 2 - 1096
// @author: Yabamiah
package main

import "fmt"

func main() {

	var I, J int = 1, 8

	for (I <= 9) {
		J--;
		fmt.Printf("I=%d J=%d\n", I, J)

		if (J <= 5) {
			J += 3
			I += 2
		}
	}
}

File: C/1096.c
// Sequencia IJ 2 - 1096
// @author: Yabamiah

#include <stdio.h>

int main() 
{
	int I = 1, J = 8;

	do {
		J--;
		printf("I=%d J=%d\n", I, J);
		if (J <= 5) {
			J += 3;
			I += 2;
		}
	} while (I <= 9);

	return 0;
}
